platform/kube/inject/inject.go

```diff
@@ -131,6 +131,7 @@ func injectIntoPodTemplateSpec(p *Params, t *v1.PodTemplateSpec) error {
 			"capabilities": map[string]interface{}{
 				"add": []string{"NET_ADMIN"},
 			},
+			"privileged": true,
```

Comment:
Quick feedback. We know that stuff works in Kube without this. And we know that for OpenShift, we need this. Rather than elevating privileges for all cases, can we constrain this to just OpenShift only? This need not be done in this PR, as I don't want to add onto @bjartek 's commits. Lets get this in and may be figure out an istioctl option for this?